dvc/repo/params/diff.py

```diff
@@ -0,0 +1,17 @@
+import dvc.utils.diff
+from .show import NoParamsError
+
+
+def _get_params(repo, *args, rev=None, **kwargs):
+    try:
+        params = repo.params.show(*args, **kwargs, revs=[rev] if rev else None)
+        return params.get(rev or "", {})
+    except NoParamsError:
+        return {}
+
+
+def diff(repo, *args, a_rev=None, b_rev=None, **kwargs):
+    old = _get_params(repo, *args, **kwargs, rev=(a_rev or "HEAD"))
+    new = _get_params(repo, *args, **kwargs, rev=b_rev)
+
+    return dvc.utils.diff.diff(old, new)
```

Comment:
This will return a dict with `diff` in it, same as with metrics. Not sure if it is okay to leave it as is and if that functionality will come in handy later. Will take a look.